src/sentry/monitors/consumers/monitor_consumer.py

```diff
@@ -880,18 +882,23 @@ def __init__(
         mode: Literal["parallel", "serial"] | None = None,
         max_batch_size: int | None = None,
         max_batch_time: int | None = None,
+        max_workers: int | None = None,
     ) -> None:
         if mode == "parallel":
             self.parallel = True
+            self.worker = ThreadPoolExecutor(max_workers=max_workers)
 
         if max_batch_size is not None:
             self.max_batch_size = max_batch_size
         if max_batch_time is not None:
             self.max_batch_time = max_batch_time
 
     def create_paralell_worker(self, commit: Commit) -> ProcessingStrategy[KafkaPayload]:
```

Comment:
Nit: `parallel`

Reply:
I'll follow up on this. I was spelling it wrong all over the place lol

Reply:
nvm just fixing here